homeassistant/components/flux_led/light.py

```diff
@@ -128,6 +134,7 @@
         ),
         vol.Optional(CONF_PROTOCOL): vol.All(cv.string, vol.In(["ledenet"])),
         vol.Optional(CONF_CUSTOM_EFFECT): CUSTOM_EFFECT_SCHEMA,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
Ok, I'll try to find some time for the refactoring. I'm quite new in home assistant-land so it might take a while. But looks like a good way to learn.. :)